apps/charterafrica/src/lib/data/common/processPageTools.js

```diff
@@ -34,9 +34,15 @@ const queryBuilder = (query) => {
 };
 
 const getRepoLink = (tool) => {
-  switch (tool.source && tool.externalId) {
+  if (!tool.externalId) {
+    return null;
+  }
+  switch (tool.source) {
     case "github":
-      return `https://github.com/${tool.externalId}`;
+      return `https://github.com/${tool.externalId.replace(
+        "https://github.com/",
+        "",
+      )}`;
```

Comment:
What is happening here?

Reply:
Most  tools have the whole url as externalID